docs/FAQ.md

```diff
@@ -10,6 +14,14 @@ We have user-contributed translations live in some languages. We intend to local
 
 If you are interested in contributing to translations please makes sure you [read this guide](how-to-translate-files.md) first.
 
+### Can I contribute articles to freeCodeCamp News or videos to freeCodeCamp's YouTube channel?
+
+Yes - you can contribute to our publication blog and YouTube channel.
+
+To write articles for freeCodeCamp news, you can follow the [publication guide here](https://www.freecodecamp.org/news/developer-news-style-guide/).
+
+To help us caption, translate, or make educational videos for our YouTube channel, you can follow the [YouTube channel guide here](https://www.freecodecamp.org/news/how-to-contribute-to-the-freecodecamp-community-youtube-channel-b86bce4c865/).
+
```

Comment:
Thanks for adding these. 

I think instead of the publication style guide which is more formal, we should point them to https://www.freecodecamp.org/news/how-to-write-for-freecodecamp - which is about what it means to be an author and introduces them to the style guide.

Maybe we can put both links here and reword it somehow to let people choose which one they want to hop into.

Reply:
I do like the idea of including both links.

Reply:
Personally, I would just update the contribute.freecodecamp.org landing page to have links to:
- How to become an author
- How to contribute to the freeCodeCamp YouTube channel

---

One thing to note: I do not think we are currently able to seek contributions towards YouTube captions.

Reply:
@ShaunSHamilton Do you know if we're currently seeking contributions towards creating YouTube videos? If not, I can omit the Youtube part entirely from this PR.

Reply:
@raisedadead @nhcarrigan Sounds great! I'll include both links.

Reply:
We are creating videos with help from the community but no longer accepting contributions to captions because it's a YouTube limitation - they have since removed the community contribution feature altogether. 

This bit about YouTube can be removed.

Reply:
Just removed the bit about captions and translations for YouTube videos in my latest commit :)